test/application/respond.js

```diff
@@ -236,7 +236,7 @@ describe('app.respond', () => {
         ctx.status = 200;
         res.setHeader('Content-Type', 'text/html');
         res.write('Hello');
-        setTimeout(() => res.end('Goodbye'), 0);
+        res.end('Goodbye');
```

Comment:
I think we should remove `res.end()` here, because this test case want `application.respond` to handle the situation that response has already been written to but not end yet.

Reply:
fixed